src/lib.rs

```diff
@@ -175,10 +175,10 @@ pub mod joints;
 pub mod stepper;
 pub mod systems;
 
-mod builder;
-mod bundle;
-mod pose;
-mod world;
+pub mod builder;
```

Comment:
These modules should not be made public, since their types are reexported.

Reply:
I got errors saying they  where too private. Confusing :)

Reply:
I haven't fixed the Amethyst example, they were originally public but I ended up just moving their types into the namespace above. Wherever they're `use`'d in Amethyst example, just remove the namespace prefix and move the type up one level.

Reply:
ah, i have a look

Reply:
Should be fixed now.